packages/visx-shape/src/shapes/Polygon.tsx

```diff
@@ -58,16 +60,17 @@ export type PolygonProps = {
 };
 
 export default function Polygon({
-  sides,
+  sides = 4,
   size = 25,
   center = DEFAULT_CENTER,
   rotate = 0,
   className,
   children,
   innerRef,
+  points = null,
```

Comment:
Removing the `null` because of the TS error

`Type 'null' is not assignable to type '[number, number][]'`

```suggestion
  points,
```